network/network_test.go

```diff
@@ -45,7 +46,7 @@ func makeTestNetwork(t *testing.T, conf *Config, opts []lp2p.Option) *network {
 	t.Helper()
 
 	net, err := newNetwork(conf, opts)
-	assert.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
@b00f 
Why use assert instead of require here? Do you know if we can use the required in the following lines?

Reply:
We can sue both. In some cases (like above) if you use assert (and the net is nil) you will get panic in the next line.
But require stops there.